stdlib/pervasives.mli

```diff
@@ -903,6 +943,11 @@ val really_input_string : in_channel -> int -> string
    characters have been read.
    @since 4.02.0 *)
 
+val input_string_all : in_channel -> string
```

Comment:
We already have the pattern of input_line, input_byte etc. We also have the odd really_input_string. Maybe input_file, implying that you're reading the whole file?

Another function I use is input_lines, which creates a list of all the lines.